src/Tasks/Microsoft.Build.Tasks.csproj

```diff
@@ -611,6 +611,10 @@
       <ExcludeFromStyleCop>true</ExcludeFromStyleCop>
     </Compile>
     <Compile Include="RCWForCurrentContext.cs" />
+    <Compile Include="..\Shared\PlatformNegotiation.cs">
```

Comment:
talked offline, but this compile item is in an itemgroup conditioned on .NET Framework

Reply:
moving this to an unconditioned item group did the trick